### DISCUSSION TITLE
Segmentation: re-reading count as a signal of loyalty

### DISCUSSION THREAD

**Discussion by adekbadek** [Automattic/newspack-popups#847] [Ideas]
Pointed out in [a CR](https://github.com/Automattic/newspack-popups/pull/233#discussion_r495677588):

> If a user visits the same page more than once, should we be recording each visit as separate activity? For certain types of pages (e.g. Liveblog) a user's repeat visits might be a true indicator of brand loyalty.

**Comment by dkoo** [comment]
This will be somewhat addressed in #869, since that refactor will now only deduplicate views of the same post or page within the same session (~1 hour).
